osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/src/main/java/org/opensmartgridplatform/adapter/protocol/iec60870/application/services/LightMeasurementDeviceResponseService.java

```diff
@@ -46,6 +48,9 @@
     @Autowired
     private LightMeasurementService lightMeasurementService;
 
+    @Autowired
+    private CorrelationUidPerDevice correlationUidQueuePerDevice;
```

Comment:
Why is there "Queue" in the name of this field? If this is a left-over from an earlier version of the code and not intentional, it is probably better to take the queue out of the name.